tests/cli/test_model_hook_order.py

```diff
@@ -67,9 +69,7 @@ def test_hook_call_order_es_mda(monkeypatch):
     test_class.setAnalysisModule = MagicMock()
     test_class.ert = MagicMock()
 
-    sim_runner_mock = MagicMock()
```

Comment:
Where these just not used?

Reply:
The mocking became a bit simpler when you are mocking the execution using the ensemble evaluator and not the legacy queue, so this intermediat mock is no longer needed, since you can specify the same return value directly 😄

Reply:
Aha 👏🏻 I forgot that you actually get a different consumption here...